vision_evaluation/evaluators.py

```diff
@@ -237,17 +260,20 @@ class F1ScoreEvaluator(EvaluatorAggregator):
     """
     F1 score evaluator for both multi-class and multi-label classification, which also reports precision and recall
     """
-
     def __init__(self, prediction_filter):
-        super().__init__([RecallEvaluator(prediction_filter), PrecisionEvaluator(prediction_filter)])
```

Comment:
those line break changes seem unnecessary and make the review hard. Can you help revert those changes first?

Line break char number is 200, per https://github.com/microsoft/vision-evaluation/blob/main/tox.ini.

Reply:
+1
Even if we decided to make a change to the style, please don't combine a feature change and style changes into one PR.